### PR TITLE
[189 pt.2] - Confirmation Modal // Tooltip text

### PR DESCRIPTION
**Pull request by W3stside**
Branched off #161

Closes #189 

Fixes wording in Confirmation modal and tooltips as laid out in #189 

Screenshot from 2021-03-08 14-53-30
Screenshot from 2021-03-08 14-53-12
Screenshot from 2021-03-08 14-53-07

### CODE REVIEW

File: src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx [hunk 1]
```diff
@@ -109,7 +110,7 @@ export default function SwapModalHeader({
               {slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(6)} {trade.outputAmount.currency.symbol}
             </b>
             {/* {' or the transaction will revert.'} */}
-            {' or the transaction will expire.'}
+            {'or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```
Comment (alfetopito) [inline src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx:113]: Keep the space

```suggestion
            {' or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```

File: src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx [hunk 2]
```diff
@@ -118,7 +119,7 @@ export default function SwapModalHeader({
               {slippageAdjustedAmounts[Field.INPUT]?.toSignificant(6)} {trade.inputAmount.currency.symbol}
             </b>
             {/* {' or the transaction will revert.'} */}
-            {' or the transaction will expire.'}
+            {'or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```
Comment (alfetopito) [inline src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx:122]: Same

```suggestion
            {' or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```

File: src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx (hunk 1 above)
Comment (alfetopito) [reply src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx:113]: screenshot_2021-03-08_07-26-37

**Comment by W3stside** [comment]
thanks @alfetopito

**Review by alfetopito — APPROVED** [review]

File: src/custom/constants/index.ts [hunk 3]
```diff
@@ -38,3 +38,5 @@ export const BUY_ETHER_TOKEN: { [chainId in ChainId]: Token } = {
 }
 
 export const ORDER_ID_SHORT_LENGTH = 8
+
+export const INPUT_OUTPUT_EXPLANATION = 'Only executed swaps incur fees.'
```
Comment (anxolin) [inline src/custom/constants/index.ts:42]: nit: This feels like overkill in my eyes. I would favour simplicity over dryness in this case

**Review by anxolin — COMMENTED** [review]
APPROVE

I didn't hit "approve" btn, so mergify don't merge automatically. Just leaving you time to either ignore my nit, or address it. If you decide to ignore, I would be fine too

File: src/custom/constants/index.ts (hunk 3 above)
Comment (anxolin) [reply src/custom/constants/index.ts:41]: okok

**Pull request merged**